### PR TITLE
feat(cli): show query language in policies when available

### PR DESCRIPTION
**Pull request by incredibler**
## How did you test this change?

* Create a policy using an existing Rego query: `/usr/local/bin/lacework policy create -f /Users/kchen/proj/opa/Rego_policy.yaml`
```
$ cat /Users/kchen/proj/opa/Rego_policy.yaml
---
  evaluatorId: Cloudtrail
  policyType: Violation
  queryId: TestRegoQueryViaCli2
  policyId: regopolicy-000
  enabled: true
  title: Test Policy for Rego Query
  description: For Rego policy testing
  remediation: |-
    Check that the console login was expected.
    AWS recommends to require MFA for all users.
  severity: high
  alertEnabled: true
  tags:
     - technique:Rego
 ```
* Show the policy:
```
$ /usr/local/bin/lacework policy show dev8-regopolicy-000
       POLICY ID        SEVERITY             TITLE               STATE    ALERT STATE   FREQUENCY                TAGS
----------------------+----------+----------------------------+---------+-------------+-----------+---------------------------------
  dev8-regopolicy-000   high       Test Policy for Rego Query   Enabled   Enabled       Hourly      technique:Rego, domain:AWS,
                                                                                                    subdomain:Configuration

                        POLICY DETAILS
--------------------------------------------------------------
    QUERY ID                      TestRegoQueryViaCli2
    POLICY TYPE                   Violation
    LIMIT                         1000
    ALERT PROFILE
    TAGS                          technique:Rego
                                  domain:AWS
                                  subdomain:Configuration
    OWNER                         kun.chen@lacework.net
    UPDATED AT                    2023-12-05T03:39:09.000Z
    UPDATED BY                    kun.chen@lacework.net
    EVALUATION FREQUENCY          Hourly
    VALID EXCEPTION CONSTRAINTS   None


        DESCRIPTION
---------------------------
  For Rego policy testing

                  REMEDIATION
------------------------------------------------
  Check that the console login was expected.

  AWS recommends to require MFA for all users.

  QUERY LANGUAGE
------------------
  Rego
                       QUERY TEXT
---------------------------------------------------------
  package lacework.policies.lacework_global_134
  import future.keywords
  import data.lacework
  import data.lacework.assessment
  source := lacework.aws.cfg.list("s3", "list-buckets")
  assess := assessment.violation(input, "just because")
```

* Show the policy in json format:
```
$ /usr/local/bin/lacework policy show dev8-regopolicy-000 --json
{
  "alertEnabled": true,
  "alertProfile": "",
  "description": "For Rego policy testing",
  "enabled": true,
  "evalFrequency": "Hourly",
  "exceptionConfiguration": null,
  "lastUpdateTime": "2023-12-05T03:39:09.000Z",
  "lastUpdateUser": "kun.chen@lacework.net",
  "limit": 1000,
  "owner": "kun.chen@lacework.net",
  "policyId": "dev8-regopolicy-000",
  "policyType": "Violation",
  "queryId": "TestRegoQueryViaCli2",
  "queryLanguage": "Rego",
  "remediation": "Check that the console login was expected.\nAWS recommends to require MFA for all users.",
  "severity": "high",
  "tags": [
    "technique:Rego",
    "domain:AWS",
    "subdomain:Configuration"
  ],
  "title": "Test Policy for Rego Query"
}
```
* list the policy with the tag `technique:Rego`
```
$ /usr/local/bin/lacework policy list --tag technique:Rego  --json
[
  {
    "alertEnabled": true,
    "alertProfile": "",
    "description": "For Rego policy testing",
    "enabled": true,
    "evalFrequency": "Hourly",
    "exceptionConfiguration": null,
    "lastUpdateTime": "2023-12-05T03:39:09.000Z",
    "lastUpdateUser": "kun.chen@lacework.net",
    "limit": 1000,
    "owner": "kun.chen@lacework.net",
    "policyId": "dev8-regopolicy-000",
    "policyType": "Violation",
    "queryId": "TestRegoQueryViaCli2",
    "queryLanguage": "Rego",
    "remediation": "Check that the console login was expected.\nAWS recommends to require MFA for all users.",
    "severity": "high",
    "tags": [
      "technique:Rego",
      "domain:AWS",
      "subdomain:Configuration"
    ],
    "title": "Test Policy for Rego Query"
  }
]
```
## Issue
https://lacework.atlassian.net/browse/RAIN-91470

### CODE REVIEW

**Comment by hazedav** [comment]
Out of curiosity, why does the policy (which refers to a query by id) have to specify the language of the query?

**Review by PengyuanZhao — APPROVED** [review]

File: api/policy.go [hunk 1]
```diff
@@ -171,24 +172,26 @@ func ParseUpdatePolicy(s string) (UpdatePolicy, error) {
 	return policy, errors.New("policy must be valid JSON or YAML")
 }
 
+type ExceptionConfigMap map[string][]PolicyExceptionConfigurationConstraints
```
Comment (incredibler) [inline api/policy.go:175]: This is to fix the linter error because of the lines below being too long

**Review by hazedav — APPROVED** [review]
Looking good bud.  It'd be awesome if we could add some integration testing for the query/policy commands as it relates to Rego (if not here perhaps in a follow up task)?

**Comment by incredibler** [comment]
> Looking good bud. It'd be awesome if we could add some integration testing for the query/policy commands as it relates to Rego (if not here perhaps in a follow up task)?

Created a task in our backlog: https://lacework.atlassian.net/browse/RAIN-91583

**Pull request merged**